docs/systemd.md

```diff
@@ -0,0 +1,55 @@
+---
+id: systemd
+title: systemd configuration
+---
+
+To manage starting, logging, etc. of Gotify, systemd can be used.
+
+# Installation
+
+For this example, Gotify is installed by root in a directory called
+`/opt/gotify/` and the executable in there is called `gotify`. The config file
+will be `/etc/gotify/config.yml` using relative paths as in the example and the
+service will be run as root. A file called `/opt/gotify.service` should be
```

Comment:
If the file needs to be in opt, then the opt path has a typo

```suggestion
service will be run as root. A file called `/opt/gotify/gotify.service` should be
```

Reply:
fixed in next commit